mutect2_workflow.nf

```diff
@@ -52,10 +61,10 @@ workflow create_split_coords_wf {
         gatk_bin_size
```

Comment:
Some spacing off here

Reply:
I think this is just displaying weird in github. It's indented 8 spaces, which is correct.